crates/kit/src/libvirt/run.rs

```diff
@@ -225,6 +225,16 @@ pub fn run(global_opts: &crate::libvirt::LibvirtOptions, opts: LibvirtRunOpts) -
         }
     }
 
+    // Display port forwarding information if any
+    if !opts.port_mappings.is_empty() {
+        println!("\nPort forwarding:");
+        for port_str in opts.port_mappings.iter() {
+            if let Ok((host_port, guest_port)) = parse_port_mapping(port_str) {
```

Comment:
I think it's better for things like this to have `port_mappings: Vec<PortMapping>` and use https://docs.rs/clap/latest/clap/_derive/_tutorial/index.html#validated-values on the `PortMapping` struct. (We could do this in other places)

Also let's not swallow errors via `if let Ok()`, I've had to fight Sonnet in other places about this ( will add to standard style across our repos soon).